### PR TITLE
Add video compression selector

### PR DESCRIPTION
**Pull request by licaon-kter**
~So, it looks ok (I've just cloned picture compression stuff), selector selects (or so it looks)... but no matter the value it ends up in [the default 360 case](https://github.com/licaon-kter/Conversations/blob/fcccf545ba21c75a4438a1522f6f134a95046cc5/src/main/java/eu/siacs/conversations/services/AttachFileToConversationRunnable.java#L108)... hence my debug logging thing there.~

~Help?!~

Fix https://github.com/siacs/Conversations/issues/3206

### CODE REVIEW

File: src/main/java/eu/siacs/conversations/services/AttachFileToConversationRunnable.java [hunk 1]
```diff
@@ -91,8 +92,22 @@ private void processAsVideo() throws FileNotFoundException {
 		mXmppConnectionService.startForcingForegroundNotification();
 		message.setRelativeFilePath(message.getUuid() + ".mp4");
 		final DownloadableFile file = mXmppConnectionService.getFileBackend().getFile(message);
-		final int runtime = mXmppConnectionService.getFileBackend().getMediaRuntime(uri);
-		final MediaFormatStrategy formatStrategy = runtime >= 20000 ? new Android360pFormatStrategy() : new Android480pFormatStrategy();
+		final MediaFormatStrategy formatStrategy;
+		final String compressVideo = mXmppConnectionService.getResources().getString(R.string.video_compression);
```
Comment (iNPUTmice) [inline src/main/java/eu/siacs/conversations/services/AttachFileToConversationRunnable.java]: This doesn’t read the preference but a translated string. The content of `compressVideo` will just be 'Video compression` or something

Comment (licaon-kter) [reply src/main/java/eu/siacs/conversations/services/AttachFileToConversationRunnable.java]: I thought this too, but logcat said "WOOMP 360 dar 360"
If I changed defaults to 720 Android Studio auto shows 720 as expected.

But then again, you the dev.... 🤷

Comment (iNPUTmice) [reply src/main/java/eu/siacs/conversations/services/AttachFileToConversationRunnable.java]: Just have a look at getCompressPicturePreference() in XmppConnectionService

Comment (licaon-kter) [reply src/main/java/eu/siacs/conversations/services/AttachFileToConversationRunnable.java]: I did, but I'll do again.

**Comment by licaon-kter** [comment]
Fixed the selector.

Limited to 360p and 720p & raised audio bitrate as per: https://developer.android.com/guide/topics/media/media-formats#video-encoding

Removed YouTube comments as they look obsolete, now they're even higher: https://support.google.com/youtube/answer/1722171?hl=en

**Pull request merged**